src/Compilers/CSharp/Test/Emit/CodeGen/CodeGenTupleTest.cs

```diff
@@ -5146,6 +5146,17 @@ public void CreateTupleTypeSymbol_ElementTypeIsError()
             Assert.Equal(SymbolKind.ErrorType, types[1].Kind);
         }
 
+        [Fact, WorkItem(13277, "https://github.com/dotnet/roslyn/issues/13277")]
+        public void CreateTupleTypeSymbol_UnderlyingTypeIsError()
+        {
+            var comp = CSharpCompilation.Create("test", references: new[] { MscorlibRef });
+
+            TypeSymbol intType = comp.GetSpecialType(SpecialType.System_Int32);
+            var vt2 = comp.CreateErrorTypeSymbol(null, "ValueTuple", 2).Construct(intType, intType);
```

Comment:
No, this is on purpose. I'm trying to do the equivalent of code that crashes the compiler in VB. See the VB test of the same name.